Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -256,7 +258,11 @@ class CreateSafeFormUIModel {
     }
 
     private func makeEthTransaction() throws -> EthTransaction {
-        try setupTransactionParameters()
+        if creationParameters == nil {
+            try setupTransactionParameters()
+        } else {
+            // No need to recreate prameters
```

Comment:
"No need to recreate transaction parameters because they are already created from creationParameters when this screen is initialized"